src/drt/src/gc/FlexGC.h

```diff
@@ -93,17 +93,8 @@ struct MarkerId
   frBlockObject* src2;
   bool operator<(const MarkerId& rhs) const
   {
-    if (box < rhs.box)
-      return true;
-    if (lNum < rhs.lNum)
-      return true;
-    if (con < rhs.con)
-      return true;
-    if (src1 < rhs.src1)
-      return true;
-    if (src2 < rhs.src2)
-      return true;
-    return false;
+    return std::tie(box, lNum, con, src1, src2)
```

Comment:
These std:: calls are likely to really hurt debug performance if this code is on the critical path. Do you know how often this method is called, and should we quantify the performance impact?

Reply:
Why do you think so?  I would expect the compiler to inline them without trouble.

Reply:
It doesn't do so in debug mode, and turns them into call instructions.

https://vittorioromeo.info/index/blog/debug_performance_cpp.html

Reply:
I missed the 'debug' bit.  I doubt it will matter but you are always welcome to profile.